README.md

```diff
@@ -287,121 +292,125 @@ Beispiel:
 
 Der `dhgefigure` Befehl wird nun auch als Snippet für Visual-Studio-Code mitgeliefert.
 
-# Abkürzungen
+# Abkürzungen und Glossar-Einträge
+
+## Abkürzungen
 
 ```latex
-\DeclareAcronym{1}{
-  short = {2},
-  long = {3}
-  }
+\newacronym{key}{short}{long}
 ```
 
-1. ID der Abkürzung, damit wird im Fließtext später referenziert.
-2. Die Abkürzung selbst
-3. Der ausgeschriebene Begriff
+1. ``key``: ID der Abkürzung, damit wird im Fließtext später referenziert.
+2. ``short``: Die Abkürzung selbst
+3. ``long``: Der ausgeschriebene Begriff
 
 Beispielweise:
 
 ```latex
-\DeclareAcronym{dhge}{
-  short = {DHGE},
-  long = {Duale Hochschule Gera-Eisenach}
-}
+\newacronym{ac:dhge}{DHGE}{Duale Hochschule Gera-Eisenach}
 ```
 
 Im Fließtext wird dann mit
 
 ```latex
-\ac{dhge}
+\gls{ac:dhge}
 ```
 
 die Abkürzung aufgerufen.
 Dies sind die Pflicht-Argumente. Es gibt weitere Einstellungsmöglichkeiten bei dem Deklarieren von Abkürzungen, die in der unten stehenden Dokumentation nachgelesen werden können. Eine sinnvolle Auswahl davon:
 
 ```latex
-\DeclareAcronym{1}{
-  short = {2},
-  long = {3},
-  short-plural = {4},
-  long-plural = {5},
-  alt = {8}
-  }
+\newacronym [
+  longplural={1},
+  shortplural={2}
+] {3}{4}{5}
 ```
 
-ODER
+1. Die ausgeschriebene Pluralform
+2. Die Pluralform der Abkürzung
+3. ID der Abkürzung, damit wird im Fließtext später referenziert
+4. Die Abkürzung selbst
+5. Der ausgeschriebene Begriff
 
-```latex
-\DeclareAcronym{1}{
-  short = {2},
-  long = {3},
-  short-plural-form = {6},
-  long-plural-form = {7},
-  alt = {8}
-  }
-```
+Die Optionen 1 und 2 sind optional.
+Beachten Sie jedoch, dass die Optionen mit einem Komma getrennt sind.
 
-1. ID der Abkürzung, damit wird im Fließtext später referenziert.
-2. Die Abkürzung selbst
-3. Der ausgeschriebene Begriff
-4. Buchstabe oder Silbe, die der Abkürzung im Plural angehangen wird
-5. Buchstabe oder Silbe, die dem ausgeschriebenen Wort im Plural angehangen wird
-6. Plural-Form der Abkürzung, ersetzt die Abkürzung komplett
-7. Plural-Form des Wortes, ersetzt das Wort komplett
-8. Alternative zum ausgeschriebenen Wort
+An einem Beispiel:
 
-Die Angaben 4 bis 8 sind optional.
+```latex
+\newacronym [
+  longplural={Jacobi-Matrizen},
+  shortplural={JMs}
+] {ac:jm}{JM}{Jacobi-Matrix}
+```
 
 Die Pluralform lässt sich mit
 
 ```latex
-\acp{1}
+\glspl{3}
```

Comment:
Muss hier nicht ein Key anstelle der 3 stehen?

Reply:
```suggestion
\glspl{key}
```

Reply:
Der "Fehler" wäre auch in Zeile 358.